app-text/ghostscript-gpl/ghostscript-gpl-9.20-r2.ebuild

```diff
@@ -235,8 +235,8 @@ src_install() {
 		fi
 	done
 
-	# install the CMaps from poppler-data properly, bug #409361
-	dosym "${EPREFIX}/usr/share/poppler/cMaps" "/usr/share/ghostscript/${PVM}/Resource/CMap"
+	# install the CMaps from poppler-data properly, bug #409361 but avoid double PREFIX-ing
+	dosym "/usr/share/poppler/cMaps" "/usr/share/ghostscript/${PVM}/Resource/CMap"
```

Comment:
Are you saying that `dosym` automatically prepends EPREFIX to the first argument? This doesn't look good.

However, in this particular case I think it'd be much better to create a relative symlinks. Could you make the patch do that instead? If you see any more absolute symlinks in the ebuild, feel free to update them as well.